src/plugin.ts

```diff
@@ -20,11 +21,13 @@ export async function plugin(inputs: PluginInputs, env: Env) {
     octokit,
     env,
     logger: {
```

Comment:
```suggestion
    logger: new Logs()
```
See also https://github.com/ubiquibot/command-start-stop/blob/c63579b364553781931f3c85515a009074c0c5e1/src/plugin.ts#L18